internal/provider/provider.go

```diff
@@ -147,7 +147,7 @@ func (p *AciProvider) Configure(ctx context.Context, req provider.ConfigureReque
 	isInsecure := stringToBool(resp, "insecure", getStringAttribute(data.IsInsecure, "ACI_INSECURE"), true)
 	validateRelationDn := stringToBool(resp, "insecure", getStringAttribute(data.ValidateRelationDn, "ACI_VAL_REL_DN"), true)
 	maxRetries := stringToInt(resp, "retries", getStringAttribute(data.MaxRetries, "ACI_RETRIES"), 2)
-	setGlobalAnnotation(data.Annotation, "ACI_ANNOTATION")
```

Comment:
think something went wrong in rebase, this removes the fix from https://github.com/CiscoDevNet/terraform-provider-aci/pull/1156

Reply:
I haven't rebased yet